scripts/veloxbench/README.md

```diff
@@ -0,0 +1,64 @@
+# Velox benchmarks
```

Comment:
This probably doesn't need to be a whole package like this, but it was easy for me to mimic how the arrow benchmarks is + having testing and other stuff there is nice